.pylintrc

```diff
@@ -1,4 +1,10 @@
+[BASIC]
+
+# Regular expression matching correct function names
+function-rgx=[a-z_][a-z0-9_]{2,50}$
```

Comment:
So we can't have function names that are 1 characters long? Aww

LGTM! :)

Reply:
Yeah, I was also thinking about 2 characters, something like `def _y():` or `def ya():` =D